packages/language-service/src/typescript_symbols.ts

```diff
@@ -308,6 +314,10 @@ class TypeWrapper implements Symbol {
   }
 }
 
+class StringIndexTypeWrappr extends TypeWrapper {
```

Comment:
```suggestion
class StringIndexTypeWrapper extends TypeWrapper {
```

Can you also add documentation for this class like you did on line 522?

Reply:
Can you fix the typo here?